### PR TITLE
Verified

### PR DESCRIPTION
**Pull request by BenediktBergmann**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Screenshot-All-Actions

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
@BenediktBergmann thank you for your submission. Our Engineer will review your request soon.

**Comment by BenediktBergmann** [comment]
@Amjed-Ayoub do you know what the current status of this is?

**Comment by Amjed-Ayoub** [comment]
@BenediktBergmann thank you for reaching out. Apologies for the delay in process. @sriyen-msft our Primary Engineer will review your request. Please expect to hear back from us soon.

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Verified/apiDefinition.swagger.json:2538]: @BenediktBergmann Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Verified/apiProperties.json
Comment (sriyen-msft) [inline certified-connectors/Verified/apiProperties.json:42]: @BenediktBergmann You agreed to not have this iconBrandColor for your connector.

image

File: certified-connectors/Verified/apiProperties.json
Comment (sriyen-msft) [inline certified-connectors/Verified/apiProperties.json:130]: @BenediktBergmann Can you please let us know why the publisher name is different from stackOwner name. Please start an email thread with us (ConnectorPartnerMgmtTeam@service.microsoft.com) and let us know how you are authorized to develop a connector "Verified AB" company.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Verified/Readme.md
Comment (BenediktBergmann) [reply certified-connectors/Verified/Readme.md:3]: Do you refer to the template mentioned on the following page?
https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-a-readme-file-artifact

I changed the title of the Action section to Operations and followed this readme
https://github.com/microsoft/PowerPlatformConnectors/tree/dev/certified-connectors/365Training

Could you please guide us/provide more information about what is missing or too much?
Aren't we allowed to have the "Good to know" part?

File: certified-connectors/Verified/apiProperties.json
Comment (BenediktBergmann) [reply certified-connectors/Verified/apiProperties.json:130]: We (CRM Konsulterna) are the partner of Verified implementing the custom connector for them.
We already have talked with MS about it and Verified has approved to MS that we are the once building it for them.
I will check whether it was send to the mentioned email.

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [reply certified-connectors/Verified/apiDefinition.swagger.json:150]: In that case, the API does not respond with any body. So the schema is actually empty. The important information is stored in the "Location" header which is provided and will be shown in the dynamic content.

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:225]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/put-envelopes-envelope-id-recipients-recipient-id

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:373]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/envelope-delete

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:406]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/put-envelopes-envelope-id

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:463]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/publishing-an-envelope

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:551]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/envelope-abort

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:679]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/documents/document-abort

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:758]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/envelope-trash

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:951]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/documents/document-delete

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:1058]: The Body is again empty and the important information will be returned in the Location header which is displayed in the dynamic values list
https://verified-dev.api-docs.io/v2.1/documents/post-envelopes-document-id-documents

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [inline certified-connectors/Verified/apiDefinition.swagger.json:1315]: As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/post-envelopes-param1-jobs-send-notification

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [reply certified-connectors/Verified/apiDefinition.swagger.json]: Should be done.

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (BenediktBergmann) [reply certified-connectors/Verified/apiDefinition.swagger.json:150]: In that case, the API does not respond with any body. So the schema is actually empty. The important information is stored in the "Location" header which is provided and will be shown in the dynamic content.
I will check the others

File: certified-connectors/Verified/Readme.md
Comment (BenediktBergmann) [reply certified-connectors/Verified/Readme.md:3]: Do you refer to the template mentioned on the following page?
https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-a-readme-file-artifact

I changed the title of the Action section to Operations and followed this readme
https://github.com/microsoft/PowerPlatformConnectors/tree/dev/certified-connectors/365Training

Could you please guide us/provide more information about what is missing or too much?
Aren't we allowed to have the "Good to know" part?

File: certified-connectors/Verified/apiProperties.json
Comment (BenediktBergmann) [reply certified-connectors/Verified/apiProperties.json]: Sorry, my fault. Should be changed now.

File: certified-connectors/Verified/apiProperties.json
Comment (BenediktBergmann) [reply certified-connectors/Verified/apiProperties.json:130]: We (CRM Konsulterna) are the partner of Verified implementing the custom connector for them.
We already have talked with MS about it and Verified has approved to MS that we are the once building it for them.
I will check whether it was send to the mentioned email.

Comment (BenediktBergmann) [reply certified-connectors/Verified/apiProperties.json:130]: There is actually a conversation where the mentioned email address is included (Subject: Verified connector)
Verified wrote on 2022-02-15 4:30 AM that we (CRM-Konsulterna) have Verifieds permission to create and manage the certification process on Verified's behalf.
@Amjed-Ayoub wrote on 2022-02-16 at 12AM (CET) that it is okay and we should go forward.

File: certified-connectors/Verified/Readme.md
Comment (sriyen-msft) [reply certified-connectors/Verified/Readme.md:3]: @BenediktBergmann The template URL needs to be updated in the documentation, You can find the latest one below:
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

You can have the "Good to know" section which is more explanation to the connector users.
And the 365Training connector was certified in the last year which is not having the latest format for readme.md

File: certified-connectors/Verified/apiDefinition.swagger.json
Comment (sriyen-msft) [reply certified-connectors/Verified/apiDefinition.swagger.json:508]: @BenediktBergmann Thanks for sharing the API documentation, You can keep the empty schema for now. 
But have a valid description value for the response code. But this may get flagged during the final review during prepping your connector production deployment. I will let you know with a resolution if it gets blocked.

File: certified-connectors/Verified/Readme.md
Comment (BenediktBergmann) [reply certified-connectors/Verified/Readme.md:3]: I now have updated the readme.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**
